frontend/src/api/error-handler.ts

```diff
@@ -0,0 +1,23 @@
+import { AxiosError } from "axios";
+import { toast } from "react-toastify";
+
+export const errorHandler = (error: AxiosError) => {
+  const { response } = error;
+  if (response && response.data) {
+    const { error } = response.data;
+    if (error) {
+      toast(error, {
```

Comment:
the 2 toasts are almost the same, extract?

Reply:
Ok, I will make a function to render the toast